apps/dashboard/config/routes.rb

```diff
@@ -49,6 +49,13 @@
     end
   end
 
+  # Jobs
+  if Rails.root.join('..', 'activejobs', 'manifest.yml').file?
```

Comment:
`Rails.root` is `apps/dashboard` dev/test or `sys/dashboard` in production. In either case, we need to look one directory above us for the old activejobs directory.